### PR TITLE
Update Montenegro holidays, add l10n support

### PR DESCRIPTION
**Pull request by KJhellico**
## Proposed change

Update Montenegro holidays, add l10n support (**cnr**, en_US, uk).

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/pin/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new `holidays` functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] I've successfully run `make check`, all checks and tests are green

[contributing-guidelines]: https://github.com/vacanza/holidays/blob/dev/CONTRIBUTING.rst
[docs]: https://github.com/vacanza/holidays/tree/dev/docs/source

### CODE REVIEW

**Review by PPsyrius — APPROVED** [review]
LGTM 🇲🇪

File: holidays/constants.py [hunk 1]
```diff
@@ -54,6 +54,7 @@
 HEBREW = "hebrew"
 HINDU = "hindu"
 ISLAMIC = "islamic"
+ORTHODOX = "orthodox"
```
Comment (arkid15r) [inline holidays/constants.py:57]: It looks like we don't use `CHRISTIAN` and for v1 `CATHOLIC` should be `CHRISTIAN_CATHOLIC`, `ORTHODOX` - `CHRISTIAN_ORTHODOX`

Or it's too much?

**Review by arkid15r — COMMENTED** [review]
LGTM

File: holidays/constants.py (hunk 1 above)
Comment (KJhellico) [reply holidays/constants.py:57]: It sounds rational. 👍

**Pull request merged**